interface/weno/adminClass.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * weno rx admin class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+class adminProperties {
```

Comment:
Working on the weno admin as requested. This PhpStorm has a high learning curve for me.  I am taking the time to correct all of the formatting issues it is pointing out.  And trying to figure out the namespace and autoloading. Not there yet.  But I can see how it will improve my coding going forward if I adhere to the warnings.